VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.ts

```diff
@@ -10,13 +9,16 @@ import { ParticipantContactDetails } from '../../shared/models/participant-conta
 })
 export class ParticipantInfoTooltipComponent {
     @Input() participant: ParticipantContactDetails;
-    constructor(private translateService: TranslateService) {}
+    joinByQuickLinkText = 'Joined by quick link';
+    quickLinkParticipantDisplayText = 'Participant';
+    quickLinkObserverDisplayText = 'Observer';
+
     getHearingRole() {
         let hearingRole = this.participant.hearingRole;
         if (this.participant.role === Role.QuickLinkParticipant) {
```

Comment:
Shouldn't this be HearingRole?

Reply:
thats a different property of the participant, participant has role and hearing role

Reply:
Ah you're right it will work for both. I was thinking we had a common role for the 2 but we do not.